helpers/dateTime.js

```diff
@@ -161,6 +161,44 @@ export function getUTCDateTimeFromLocalDateTime(date, time) {
 	return formatDateTimeInISO(utcDateTime);
 }
 
+export function getUTCDateTimeRange(type, diff) {
+	if (!type || (!diff && diff !== 0)) return {};
```

Comment:
Would positive and negative values be valid?

Reply:
With the current implementation no, since negative values would mix up which date would be `startValue` and which would be `endValue`. I'll look into improving this though or at least making it more clear how to use this.

Reply:
Currently for `lastHour`, filter calls: 
```javascript
const dateTimeRange = getUTCDateTimeRange('hours', 1);
```
But maybe the consumer should use negative for ranges in the past, and positive for ranges in the future.
```javascript
// lastHour
const dateTimeRange = getUTCDateTimeRange('hours', -1);

// nextHour
const dateTimeRange = getUTCDateTimeRange('hours', 1);
```
I know filter doesn't have the `nextHour` case, but perhaps we should set this helper to be more generic in that sense, since it has potential to be used by anything. That's a decision we'd want to make now.

This helper assumes the reference date is _now_. Maybe that's ok. In the future we would accept option for the consumer to specify the reference date if necessary.

Thoughts?

Reply:
I agree - that seems much more flexible and future proof. Will update